README.md

```diff
@@ -166,20 +168,21 @@ Note that since a non-activated portal has no storage access, it cannot join any
 
+- Portals are top-level browsing contexts, and given their own [browsing context group](https://html.spec.whatwg.org/multipage/browsers.html#groupings-of-browsing-contexts). This means they cannot synchronously access their embedder, e.g. to navigate it or modify its DOM (even when same-origin).
+
 - Any features that are controlled by the [Permissions API](https://w3c.github.io/permissions/) ([list](https://w3c.github.io/permissions/#permission-registry)) will be automatically denied without prompting.
 
 - Any features controlled by [Permissions Policy](https://w3c.github.io/webappsec-permissions-policy/) ([list](https://github.com/w3c/webappsec-permissions-policy/blob/master/features.md)) will be disabled, unless their default allowlist is `*`. There is no ability for the host page to delegate these permissions. (In particular, there is no counterpart to `<iframe>`'s `allow=""` attribute.)
 
-- If `window.open()` is used to open a new window (instead of navigate an `<iframe>` inside the portaled content), then it will instead do nothing and return `null`.
+- Popups, pointer lock, orientation lock, the presentation API, downloads, and modal dialogs (`alert()` etc.) all are disabled pre-activation. (These are features which are currently only possible to disable with through [iframe sandboxing](https://html.spec.whatwg.org/multipage/origin.html#sandboxing).)
 
 After activation, these restrictions are lifted: the portaled content is treated like a normal top-level browsing context, and is able to use all these features in the normal way. The `portalactivate` event can be used to request permissions upon activation, if necessary. (Although doing so gives a user experience equivalent to requesting permissions on load, and thus is rarely the best design.)
 
+_Note: ideally Permissions Policy would become a superset of the Permissions API and iframe sandboxing. Then we could use that infrastructure as the single place to impose and lift these restrictions. Currently that is not the case, so the spec will be more messy._
+
 TODO:
 
-- Do these restrictions also apply to same-origin portals?
-- Discuss CSP, document policies, `sandbox=""`, referrer policy... any other policies? Might not belong in this section.
-- Briefly discuss permissions and policies that people might want to impose on portals, including some that aren't proposed yet (like blocking scripts or network access). Note that these will be developed independently, for iframes and portals alike.
-- Ideally Permissions Policy would become a superset of the Permissions API, and we could use that infrastructure exclusively to shut things down. Currently that is not the case so we have to consult both.
+- Do all of these restrictions also apply to same-origin portals? I lean towards "yes"?
```

Comment:
I also lean towards "yes", though for same-origin it might be more reasonable to allow permission policy delegation in the future (insofar as the origin pair for the permission grant is the same etc, so keeping the permission across activation is sensical), subject to the discussions @a4sriniv had with permissions folk. But I'm not aware of that being a priority so punting seems cool.

Reply:
Let's make it so for now.